File: fizzbuzz.py
#!/usr/bin/python

count = 0
while (count <= 100):
    fizz = 'fizz' if count % 3 == 0 else ''
    buzz = 'buzz' if count % 5 == 0 else ''
    print count,' ',fizz,buzz
    count = count + 1

File: fizzbuzz.go
package main

import "fmt"

func main() {

	var fizz string
	var buzz string

	for n := 0; n <= 100; n++ {
		str := "%d %s%s"
		if n%3 == 0 {
			fizz = "fizz"
		} else {
			fizz = ""
		}
		if n%5 == 0 {
			buzz = "buzz"
		} else {
			buzz = ""
		}
		res := fmt.Sprintf(str, n, fizz, buzz)
		fmt.Println(res)
	}
}
